modules/tests/jvm/src/test/scala/SslTestPlatform.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018-2021 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package tests
+
+import cats.effect._
+import skunk._
+import natchez.Trace.Implicits.noop
+
+trait SslTestPlatform { self: SslTest =>
```

Comment:
Why are these tests not run for JS?

Reply:
So you just have to tell node to ignore EVERY certificate? You can't do it per connection? That's kind of nuts.

Reply:
I think you can do it on the _global_ level with the environment variable and possibly on the _socket_ level by fiddling with the TLS parameters: https://github.com/typelevel/fs2/blob/main/io/js/src/main/scala/fs2/io/net/tls/TLSParameters.scala#L44

But, no way to do it for an entire context.